packages/framework/src/Model/Product/Elasticsearch/ProductExportRepository.php

```diff
@@ -378,15 +378,19 @@ protected function getBrandUrlForDomainByProduct(Product $product, int $domainId
 
     /**
      * @param \Shopsys\FrameworkBundle\Model\Product\Product $product
+     * @param int $domainId
+     * @param int|null $limit
      * @return array
      */
-    protected function extractAccessoriesIds(Product $product): array
+    protected function extractAccessoriesIds(Product $product, int $domainId, ?int $limit): array
```

Comment:
I don't think it's necessary to make changes in this class at all... 
in the end we only pass the same value as already is as the default value in productAccessorFacade